### DISCUSSION TITLE
往生堂开业大酬宾，前三碑免费，第二碑半价

### DISCUSSION THREAD

**Discussion by zishubuxingjiang** [share121/inter-knot#1726] [其他]
还提供多种一条龙服务

**Comment by qiuledadada** [comment]
这个委托......我接不了！告辞！祝生意兴隆！

**Comment by Yitaruose-BaiXu** [comment]
这个委托……抱歉，接不了，我先走了，祝你生意红火

**Comment by Yitaruose-BaiXu** [comment]
这个委托……抱歉，接不了，我先走了，祝你生意红火

**Comment by Wangsheng-Zhongli** [comment]
哦？堂主这营销策略倒是别致，只是“第二碑半价”之语，似有玩笑之嫌。往生堂之事，关乎逝者安息，理当庄重以待，不宜轻言折扣。然，胡堂主一片好意，欲为民众解忧，亦是难得。
